### PR TITLE
Update Checkbox documentation section about side-loading providers

### PR DESCRIPTION
**Pull request by pieqq**
## Description

Documentation update to refresh the ["Side-loading Providers" page.](https://checkbox.readthedocs.io/en/latest/side-loading.html)

### CODE REVIEW

File: checkbox-ng/docs/side-loading.rst [hunk 1]
```diff
@@ -48,26 +47,28 @@ Create ``checkbox-providers`` directory in ``/var/tmp/``::
...
 .. tip::
     --depth=1 tells git not to download all the history of the repo
 
 When started, Checkbox should display following warning::
 
-    $ checkbox-snappy.checkbox-cli
+    $ checkbox.checkbox-cli
     WARNING:plainbox.session.assistant:Using side-loaded provider:
-    com.canonical.certification:plainbox-provider-checkbox
+    com.canonical.certification:checkbox-provider-base
```
Comment (p-gentili) [inline checkbox-ng/docs/side-loading.rst:64]: Just tried, the print seems a bit different:
```
ubuntu@ubuntu-HP-ProBook-440-14-inch-G9-Notebook-PC:/var/tmp$ checkbox-cli
Using sideloaded provider: checkbox-provider-base, version 2.1.0 from /var/tmp/checkbox-providers/base
```

File: checkbox-ng/docs/side-loading.rst [hunk 2]
```diff
@@ -48,26 +47,28 @@ Create ``checkbox-providers`` directory in ``/var/tmp/``::
     You may not have write permissions for ``/var/tmp/``. You may want to
     run mkdir with sudo and later ``chown`` that directory
 
-Clone plainbox-provider-checkbox to the side-loaded directory::
+Clone Checkbox repository and copy the base provider to the side-loaded
+directory::
```
Comment (p-gentili) [inline checkbox-ng/docs/side-loading.rst:51]: double ":" at the end

**Review by p-gentili — CHANGES_REQUESTED** [review]
I've recently used side-loading and I can confirm everything is OK here (except for a couple of small things to fix).

File: checkbox-ng/docs/side-loading.rst (hunk 1 above)
Comment (pieqq) [reply checkbox-ng/docs/side-loading.rst]: Ah, nice! I had just manually updated the content, but you're right, with more recent versions of Checkbox, you get a different message. Thanks!

**Pull request merged**